test/standardrb_test.rb

```diff
@@ -30,7 +29,6 @@ def test_project_with_cwd_in_nested_path
     refute status.success?
     assert_same_lines <<-MSG.gsub(/^ {6}/, ""), stdout
       standard: Use Ruby Standard Style (https://github.com/testdouble/standard)
-      standard: Run `standardrb --fix` to automatically fix some problems.
```

Comment:
These lines were changed because this seemed the most "high-level" test of the suite and, therefore, should present the most authentic response possible.